backend/attendance-service/src/main/java/org/egov/util/AttendanceServiceConstants.java

```diff
@@ -3,12 +3,12 @@
 public class AttendanceServiceConstants {
     public static final String MASTER_TENANTS = "tenants";
```

Comment:
@vguptatw - shouldn't we get roles from roles.json in MDMS? The risk here is that if the role codes change there, we will have to keep updating here.